src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TaskController.java

```diff
@@ -0,0 +1,71 @@
+package com.matuageorge.technicalmaintenanceschedule.controller;
+
+import com.matuageorge.technicalmaintenanceschedule.dto.TaskDto;
+import com.matuageorge.technicalmaintenanceschedule.exception.NotFoundException;
+import com.matuageorge.technicalmaintenanceschedule.exception.ResourceAlreadyExistsException;
+import com.matuageorge.technicalmaintenanceschedule.exception.ValidationException;
+import com.matuageorge.technicalmaintenanceschedule.model.Task;
+import com.matuageorge.technicalmaintenanceschedule.service.TaskService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@CrossOrigin
+@RestController
+@Slf4j
+@AllArgsConstructor
+@RequestMapping("/tasks")
+public class TaskController {
+
+    private final TaskService taskService;
+
+    @PostMapping("/admin")
+    public ResponseEntity<Task> save(@RequestBody TaskDto taskDto) throws ValidationException, ResourceAlreadyExistsException {
+
+        log.info("Handling task {} to create", taskDto);
+
+        Task savedTask = taskService.save(taskDto);
```

Comment:
you can inline this

Reply:
please read article about naming conventions in rest. and about rest principles.https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/
Endpoint tasks/admin say nothing about what is going on.

Reply:
ok